lib/modules/apostrophe-templates/index.js

```diff
@@ -411,13 +411,14 @@ module.exports = {
       // Format the given date with the given momentjs
       // format string.
 
-      env.addFilter('date', function(date, format) {
+      env.addFilter('date', function(date, format, locale='en') {
```

Comment:
Please avoid ES6 in this version of Apostrophe, you can write:

`locale = locale || 'en'`

Inside the function.

Reply:
Added the requested changes